components/Avatar/avatar.css

```diff
@@ -0,0 +1,45 @@
+.avatar{
+    height:5rem;
+    border-radius: 50%;
+    margin: 1rem;
+   cursor: pointer;
+}
+.avatar-content{
+    display: flex;
+    
+}
+.avatar-area{
+    margin: 0rem;
+}
+.md{
+    width: 7rem;
+    height:7rem;
+}
+.lg{
+    width: 9rem;
+    height:9rem;
+}
+@media screen and (max-width: 600px){
+    .avatar-content{
+        display: flex;
+        flex-direction: column;
+    }
+    .avatar{
+        width: 4rem;
+        height:4rem;
+        border-radius: 50%;
+        margin: 1rem;
+       cursor: pointer;
+    }
+    .avatar-area{
+        margin: 0rem;
+    }
+    .md{
```

Comment:
You don't have tp define same properties in same classes, while writing the media queries. 
Redefine only those properties/ style classes, which will be changed, when the resizing takes place.